### PR TITLE
Adds account_id to channel create (and update), missing functions

### PR DESCRIPTION
**Pull request by tzarebczan**
- Adds account id to channel create (And update via same params)
- We were also missing transaction list / utxo release

### CODE REVIEW

File: extras/jsonrpc/daemon_types.go [hunk 1]
```diff
@@ -471,6 +471,50 @@ type UTXOListResponse []struct {
 	Type          string `json:"type"`
 }
 
+type UTXOReleaseResponse *string
```
Comment (sayplastic) [inline extras/jsonrpc/daemon_types.go]: Can just use a `string` here.

Comment (tzarebczan) [reply extras/jsonrpc/daemon_types.go]: Wasn't sure if that would handle this null response from the sdk: 

{
  "jsonrpc": "2.0",
  "result": null
}

Comment (nikooo777) [reply extras/jsonrpc/daemon_types.go]: that might be problematic indeed. @eukreign why is the SDK returning null? Could this be improved?

File: extras/jsonrpc/daemon_types.go [hunk 2]
```diff
@@ -471,6 +471,50 @@ type UTXOListResponse []struct {
...
+
+type TransactionListResponse []struct {
```
Comment (nikooo777) [inline extras/jsonrpc/daemon_types.go]: there's a lot of code reuse here. I think I can clean this up by reusing the structures we already have

File: extras/jsonrpc/daemon_types.go (hunk 1 above)
Comment (eukreign) [reply extras/jsonrpc/daemon_types.go]: there is no return value from this function, i guess somewhere down the line `null` gets filled in as a default instead, why is it problematic? `go` doesn't support calling RPC with no result?

**Pull request closed without merging**